designs/UI/business_creation.html

```diff
@@ -0,0 +1,110 @@
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g Samwel Joyk" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Email Address</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g samwelemail@gmail.com" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+
+                        <div class="form-group">
+                                <div class="text-left" style="">Phone number</div>
+                                <div class="input-group">
+                                    <div class="input-group-prepend">
+                                    </div>
+                                    <input type="text" class="form-control" placeholder="e.g +25471709999" aria-label="samsonanami" aria-describedby="basic-addon1">
+                                  </div>
+                        </div>
+                        
+                        <div class="text-left" style="">Select Logo</div>
+                        <div class="form-group">
+                            <div class="input-group">
+                                <input type="file" class="form-control" id="lastname" placeholder="password">
+                            </div>
+                        </div>
+
+                        
+                        <div class="form-group">
+                            <div class="col-sm-offset-2 col-sm-10">
+                              
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <div class="text-center">
+                                <button type="submit" class="button button4 btn-primary">Create Business >></button>
+                            </div>
+                        </div>
+                       </form>
+                </div>
+                </div>
+            </div>
+            <div class="col-xs-6 col-sm-2"></div>
+        </div>
+    </div>
+    
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://code.jquery.com/jquery.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files
+     as needed -->
+    <script src="js/bootstrap.min.js"></script>   
+</body>
+</html>
```

Comment:
Add a new line at the end of your file

Reply:
ok, done also to the rest of the files. thanks